lib/db.js

```diff
@@ -91,6 +106,34 @@ DB.prototype.getTransactionWithBlockInfo = function(txid, queryMempool, callback
   });
 };
 
+DB.prototype.sendTransaction = function(tx, callback) {
+  if(tx instanceof this.Transaction) {
+    tx = tx.toString();
+  }
+  $.checkArgument(typeof tx === 'string', 'Argument must be a hex string or Transaction');
+
+  try {
+    var txid = this.bitcoind.sendTransaction(tx);
+    return callback(null, txid);
+  } catch(err) {
+    return callback(err);
+  }
+};
+
+DB.prototype.estimateFee = function(blocks, callback) {
+  var self = this;
+
+  // For some reason getting fee for 1 block returns -1
+  // Until this is resolved, just make it 2 blocks
```

Comment:
Yeah, I'm wondering why that behavior exists. On testnet `-1` will be returned for 2+ and higher.